pkg/controller/build/image_build_request.go

```diff
@@ -274,7 +201,7 @@ func (i ImageBuildRequest) toPodmanPod() *corev1.Pod {
 		},
```

Comment:
suggestion: Lets get rid of this `toPodmanPod()` method for the time-being. It is not being used anywhere and is only taking up space. Its existence is probably better referred to by looking through Git history as opposed to it being there all the time :smile:.